rust/windows-client/firezone-windows-common/README.md

```diff
@@ -0,0 +1,17 @@
+firezone-windows-common
```

Comment:
I'd prefer if we use `connlib-shared` rather than adding a new crate.

Reply:
we can add a windows specific module

Reply:
Cool, I'll revise this PR